### ISSUE TITLE
Unfree files are included in a few Wikipedia ZIM files

### ISSUE THREAD

**Issue by kelson42** [openzim/zim-requests#167]
For example:
wikipedia_it_all_01_2013 contains https://it.wikipedia.org/wiki/File:Bonnie_Rockwaller.jpg

**Comment by Popolechien** [comment]
Well there's also a ton of fair use content on enwp, logos for a start but also low-resolution illustrations, e.g. https://en.wikipedia.org/wiki/File:Super_Mario_Bros._box.png

**Comment by kelson42** [comment]
@Popolechien I'm not certain this is a good idea to invoke - in Switzerland - the concept of fear use... in particular for all potential usage which can be done with our Wikipedia ZIM files.

**Comment by Popolechien** [comment]
I agree, but I'd rather comply with a takedown request than preemptively nuke thousands of images out of our zim files (with the dev costs associated). This being said, better be safe than sorry and prepare already for the day it happens.

**Comment by Popolechien** [comment]
Also, shouldn't this be a request for /mwoffliner?

**Comment by kelson42** [comment]
@Popolechien yes, will transfer it.
